### PR TITLE
Simplify RBACs for pipeline-service

### PR DESCRIPTION
**Pull request by Roming22**
@manish-jangra is trying to sanitize the RBACs in the repository.

I can't recall why we decided to have 2 sets of roles, but at the moment I believe we need just one for the team that grants us access to all tekton related resources, and gives us admin rights on the namespace we manage (so we can restart pods for example).

@sbose78 @jduimovich Please let me know if you do not agree with the content of this PR.

### CODE REVIEW

**Review by sbose78 — APPROVED** [review]

**Comment by sbose78** [comment]
If its a small group with access, this should suffice.

**Comment by manish-jangra** [comment]
looks good to me if you want to keep one cluster role 👍

File: components/pipeline-service/base/rbac/cluster-role/pipeline-service-sre.yaml [hunk 1]
```diff
@@ -1,62 +1,31 @@
...
-  - eventlisteners
-  - triggerbindings
-  - triggers
-  - triggertemplates
+  - "*"
   verbs:
-  - get
-  - list
-  - watch
   - create
```
Comment (manish-jangra) [inline components/pipeline-service/base/rbac/cluster-role/pipeline-service-sre.yaml:23]: need `create` access? Shouldn't that be handled by ArgoCD?

**Review by manish-jangra — CHANGES_REQUESTED** [review]
need `create` access for tekton resources? Shouldn't that be handled by ArgoCD?

**Comment by Roming22** [comment]
Force merge.

The content has been validated multiple times by now, and there's no reason for it to impact the tests since the changes are non-functional.

**Pull request merged**